src/components/Cars/Cars.jsx

```diff
@@ -0,0 +1,43 @@
+export const Cars = ({ sortedCars, rowHeadings }) => (
+  <table>
+    <thead>
+      <tr>
+        {rowHeadings.map(heading => (
```

Comment:
It would be great to create a separate component for table, and a separate component for Car row